lib/manager/gradle-wrapper/artifacts.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint jest/no-standalone-expect: 0 */
```

Comment:
Should be fixed, as `expect` does not work at all inside `describe`

docs: [no-standalone-expect-in-a-describe-block-no-standalone-expect](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-standalone-expect.md#no-standalone-expect-in-a-describe-block-no-standalone-expect)